pyproject.toml

```diff
@@ -40,7 +40,7 @@ cosl = ">= 0.0.5"
 tenacity = ">=8.0.1"
 pure-sasl = ">=0.6.2"
 jsonschema = ">=4.10"
-cryptography = "^39.0.0"
+cryptography = ">39.0.0"
```

Comment:
**question/thought:** Non-blocking, are we doing `>` here because we almost always have to update cryptography because of some security issue?

If so, this is probably fine, but before they switched to Rust, when they did switch having a `>` would have borked everything, so just worth keeping an eye on this

Reply:
I'm not sure tbh. I think this comes from a diff with some older version, or maybe the k8s code. I can switch it back.

Reply:
No need, doesn't seem to be harmful!